source/puppet/4.2/reference/release_notes.markdown

```diff
@@ -20,19 +17,91 @@ Make sure you also read the [Puppet 4.0 release notes](/puppet/4.0/reference/rel
 
 Also of interest: the [Puppet 4.1 release notes](/puppet/4.1/reference/release_notes.html).
 
+## Puppet 4.2.3
+
+Released October 29, 2015
+
+Shipped in puppet-agent version 1.2.7.
+
+Puppet 4.2.3 is a bug fix and platform support release in the Puppet 4.2 series. It also removes support for Ubuntu 14.10 (Utopic).
+
+For JIRA issues related to Puppet 4.2.3, see:
+
+* [Fixes for Puppet 4.2.3]()
+* [Introduced in Puppet 4.2.3]()
+
+### New Platforms: AIX 5.3, 6.1, and 7.1, and Solaris 11
+
+Puppet 4.2.3 supports AIX 5.3, 6.1, and 7.1 on PowerPC architectures, and Solaris 11 on SPARC and i386 architectures.
+
+### SUPPORT END: End Support for Ubuntu 14.10 (Utopic)
+
+Ubuntu 14.10 (Utopic) reached the end of its life on [July 23, 2015](https://lists.ubuntu.com/archives/ubuntu-announce/2015-July/000198.html). Support for it ends with Puppet 4.2.3. We continue to support Ubuntu 14.04 (Trusty) and 15.04 (Vivid).
+
+* [PUP-5001](https://tickets.puppetlabs.com/browse/PUP-5001)
+
+### SECURITY: Access CA Certificates Only from a Secure Directory in Windows
+
+In previous versions of Puppet 4 for Windows, the OpenSSL component of puppet-agent attempts to load Certificate Authority (CA) certificates from an insecure, world-writable directory. Puppet 4.2.3 changes the OpenSSL component to load CA certificates only from a secure directory (`C:\Windows\System32\ssl`).
+
+* [PUP-5215](https://tickets.puppetlabs.com/browse/PUP-5215)
```

Comment:
This is actually a puppet-agent ticket https://tickets.puppetlabs.com/browse/PA-27 (in the new world of building puppet--agent packages). The other PUP ticket 5215 was for puppet 3.x and will be released in an upcoming puppet 3.8.4 release.